javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -76,6 +76,22 @@ public void testJvmFlags() {
     org.hamcrest.MatcherAssert.assertThat(System.getProperty("foo.bar"), is("true"));
   }
 
+  @Test
+  public void testExternalDeps() throws Exception {
+    loadIntegrationTestRuleIntoWorkspace();
+
+    driver.scratchFile(
```

Comment:
where is the assertion here?

Reply:
Nowhere, this test is a no-op and should not be here, sorry for that.